location.md

```diff
@@ -36,13 +36,19 @@ With three state universities, many tech companies ranging from small startups t
 
 ## Travel
 
-<!-- TODO: Change me -->
-Zürich is well connected to the rest of Europe with many direct flights and trains to other major European cities every day.
+### By train
+
+Paris is very easily reachable by train. There are [seven different train stations](https://en.wikivoyage.org/wiki/Paris#By_train), all of which are connected through the local transport system. Many direct trains are available from various European cities such as Amsterdam, Brussels, Cologne, Frankfurt, London, Milan, Munich, Turin, and Zurich.
+
+### By plane
+
+Paris has two international airports, **CDG** and **ORY**. The main airport, [Paris Charles de Gaulle Airport (**CDG**)](https://en.wikivoyage.org/wiki/Paris_Charles_de_Gaulle_Airport), is located in the north of Paris. This airport is connected with central Paris through train line *RER B*.
+
+The second, older airport, [Paris Orly Airport (**ORY**)](https://en.wikivoyage.org/wiki/Paris_Orly_Airport), is located southwest of the city. The *Orlyval* light train connects the airport to *Antony* station, which is part of the *RER B* line. Alternatively, the cheaper *OrlyBus* connects to *Denfert-Rochereau* station.
```

Comment:
I clearly recommend the bus over the direct train line. The train line is very expensive and the ticket is only working on it, not on the global network.